Apps/Sandcastle/gallery/glTF PBR Extensions.html

```diff
@@ -0,0 +1,260 @@
+        const environmentMapURL =
+          "https://cesium.com/public/SandcastleSampleData/kiara_6_afternoon_2k_ibl.ktx2";
+
+        // To generate the spherical harmonic coefficients below, use Google's Filament project:
+        // 1 - Download the Filament release (https://github.com/google/filament/releases).
+        // 2 - Run `cmgen --type=ktx --deploy=/path/to/output /path/to/image.hdr`. Other formats are also supported. Run `cmgen --help` for all options.
+        // 3 - Take the generated coefficients and load them in CesiumJS as shown below.
+        const L00 = new Cartesian3(
+          1.234709620475769,
+          1.221461296081543,
+          1.273156881332397
+        );
+        const L1_1 = new Cartesian3(
+          1.135921120643616,
+          1.171217799186707,
+          1.287644743919373
+        );
+        const L10 = new Cartesian3(
+          1.245193719863892,
+          1.245591878890991,
+          1.282818794250488
+        );
+        const L11 = new Cartesian3(
+          -1.106930732727051,
+          -1.112522482872009,
+          -1.153198838233948
+        );
+        const L2_2 = new Cartesian3(
+          -1.086226940155029,
+          -1.079731941223145,
+          -1.101912498474121
+        );
+        const L2_1 = new Cartesian3(
+          1.189834713935852,
+          1.185906887054443,
+          1.214385271072388
+        );
+        const L20 = new Cartesian3(
+          0.01778045296669,
+          0.02013735473156,
+          0.025313569232821
+        );
+        const L21 = new Cartesian3(
+          -1.086826920509338,
+          -1.084611177444458,
+          -1.111204028129578
+        );
+        const L22 = new Cartesian3(
+          -0.05241484940052,
+          -0.048303380608559,
+          -0.041960217058659
+        );
+        const coefficients = [L00, L1_1, L10, L11, L2_2, L2_1, L20, L21, L22];
+
+        const imageBasedLighting = new Cesium.ImageBasedLighting({
+          luminanceAtZenith: 0.7,
+          sphericalHarmonicCoefficients: coefficients,
+          specularEnvironmentMaps: environmentMapURL,
+        });
+
+        const lightingOptions = [
+          {
+            text: "Environment map lighting",
+            onselect: () => {
+              imageBasedLighting.sphericalHarmonicCoefficients = coefficients;
+              imageBasedLighting.specularEnvironmentMaps = environmentMapURL;
+            },
+          },
+          {
+            text: "Procedural sky lighting",
+            onselect: () => {
+              imageBasedLighting.sphericalHarmonicCoefficients = undefined;
+              imageBasedLighting.specularEnvironmentMaps = undefined;
+            },
+          },
+        ];
+        Sandcastle.addToolbarMenu(lightingOptions, "lightingToolbar");
+
+        let model;
+        const modelOptions = [
+          {
+            text: "Specular Test",
```

Comment:
Could we start with the Clear Coat Wicker model? It matches the thumbnail, and also provides a more "real life" use case as compared with the test models.